Jupyter Panel Preview not working on `main`

I am on the main branch. I would like to take and contribute a few screenshots I think are missing in the documentation.

I would like one using the Panel Jupyter Preview. But it does not work for me. I'm on the `main` branch of Panel. I'm running inside my `panel_dev` python environment. And that is also where Jupyterlab is installed and started from.

![image](https://github.com/holoviz/panel/assets/42288570/b3bc3400-add6-490f-a728-123611551328)

```bash
404 : Not Found
You are requesting a page that does not exist!
```

My version of Jupyterlab is 3.6.3.

I just tried it on my Jupyterhub. And its not working there either.

The JupyterHub is running

panel==0.14.4 bokeh==2.4.3

My panel_dev environment is running

panel==1.0.0rc7, bokeh==3.1.1

![image](https://github.com/holoviz/panel/assets/42288570/fe84ffbc-a9b9-4680-bf5d-a9c983ee6927)

```bash
Uncaught Error: Cannot find module '@bokehjs/models/layouts/alignments'
```

--- a/panel/io/jupyter_server_extension.py
+++ b/panel/io/jupyter_server_extension.py
@@ -111,7 +111,7 @@ import os
 import pathlib
 import sys
 
-app = '{{ path }}'
+app = r'{{ path }}'
 os.chdir(str(pathlib.Path(app).parent))
 sys.path = [os.getcwd()] + sys.path[1:]
 
@@ -186,6 +186,8 @@ class PanelJupyterHandler(JupyterHandler):
                 comm_id = msg['content']['comm_id']
             elif msg_type == 'stream' and msg['content']['name'] == 'stderr':
                 logger.error(msg['content']['text'])
+            elif msg_type == "error":
+                logger.error(msg['content'])
         return result, comm_id, extension_dirs
 
     @tornado.web.authenticated

--- a/panel/io/jupyter_server_extension.py
+++ b/panel/io/jupyter_server_extension.py
@@ -187,7 +187,7 @@ class PanelJupyterHandler(JupyterHandler):
             elif msg_type == 'stream' and msg['content']['name'] == 'stderr':
                 logger.error(msg['content']['text'])
             elif msg_type == "error":
-                logger.error(msg['content'])
+                logger.error(msg['content']['traceback'])
         return result, comm_id, extension_dirs
 
     @tornado.web.authenticated

